decidim-admin/spec/commands/create_feature_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+module Decidim
+  module Admin
+    describe CreateFeature do
+      let(:manifest) { Decidim.features.find{ |feature| feature.name == :dummy }}
```

Comment:
This and similar code is used in different places, maybe we could add finder methods to `Decidim`